### ISSUE TITLE
Support for heatmaps

### ISSUE THREAD

**Issue by henrikjohansen** [Graylog2/graylog2-web-interface#531]
Heatmaps are awesome for visualising lots of data points to spot trends and patterns - see http://queue.acm.org/detail.cfm?id=1809426 for more info :)

**Comment by fadenb** [comment]
Is this feature on the roadmap for the near future?

**Comment by kroepke** [comment]
No, we currently do not have this on the roadmap. However, visualization types are pluggable since 2.0, so it would be possible to add it to Graylog without core changes. Most of the difficulty would probably be in preparing the data (and doing the queries) in a way that is meaningul for generating heatmaps.
